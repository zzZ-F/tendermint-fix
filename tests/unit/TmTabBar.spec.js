import { mount } from "@vue/test-utils"
import { TmTabBar } from "@/index"

describe("TmTabBar", () => {
  let wrapper

  beforeEach(() => {
    wrapper = mount(TmTabBar, { slots: { default: "<div>slot</div>" } })
  })

  it("has the expected html structure", () => {
    expect(wrapper.vm.$el).toMatchSnapshot()
  })
})
