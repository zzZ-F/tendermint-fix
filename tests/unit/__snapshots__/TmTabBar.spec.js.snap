// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TmTabBar has the expected html structure 1`] = `
<div
  class="tm-tab-bar"
>
  <div
    class="tm-tab-bar-container"
  >
    <div>
      slot
    </div>
  </div>
</div>
`;
